crates/rover-error/src/metadata/suggestion.rs

```diff
@@ -20,6 +26,30 @@ impl Display for Suggestion {
                     Yellow.normal().paint("'--sensitive'")
                 )
             }
+            Suggestion::SetConfigHome => {
+                format!(
+                    "You can override this directory by setting the {} environment variable.",
+                    Yellow.normal().paint(RoverEnvKey::ConfigHome.to_string())
+                )
+            }
+            Suggestion::MigrateConfigHomeOrCreateConfig => {
+                format!("If you've recently changed the {} environment variable, you may need to migrate your old configuration directory to the new path. Otherwise, try setting up a new configuration profile by running {}",
```

Comment:
```suggestion
                format!("If you've recently changed the {} environment variable, you may need to migrate your old configuration directory to the new path. Otherwise, try setting up a new configuration profile by running {}.",
```